Directory.Build.props

```diff
@@ -11,16 +11,19 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <FhirServerPackageVersion>2.0.55</FhirServerPackageVersion>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <HealthcareSharedPackageVersion>6.1.145</HealthcareSharedPackageVersion>
+    <HealthcareSharedPackageVersion>6.2.0</HealthcareSharedPackageVersion>
     <HighEntropyVA>true</HighEntropyVA>
     <Hl7FhirPackageVersion>4.3.0</Hl7FhirPackageVersion>
     <IdentityModelTokenPackageVersion>6.25.0</IdentityModelTokenPackageVersion>
     <LangVersion>Latest</LangVersion>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <NetStandardVersion>netstandard2.0</NetStandardVersion>
+    <NetVersion>net7.0</NetVersion>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Product>Microsoft Health</Product>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <SdkPackageVersion>6.0.0</SdkPackageVersion>
+    <SdkPackageVersion>7.0.0</SdkPackageVersion>
+    <TargetFramework>$(NetVersion)</TargetFramework>
```

Comment:
We do need to keep .NET 6 for the Azure Functions, as they don't support .NET 7. So we'll need the same code that choses `<SdkPackageVersion>` based on the target framework. It also means we'll need to multiple target any package used by functions.